cmd/ocm-backplane/cloud/assume.go

```diff
@@ -186,3 +208,23 @@ func runAssume(_ *cobra.Command, args []string) error {
 	}
 	return nil
 }
+
+type Environment string
+
+func (e *Environment) String() string {
+	return string(*e)
+}
+
+func (e *Environment) Set(env string) error {
+	switch strings.ToLower(env) {
+	case "int", "stg", "prod":
```

Comment:
This may not always true

Reply:
That's fair. Though we can change this as needed.